### DISCUSSION TITLE
Visualize b/y ions

### DISCUSSION THREAD

**Discussion by Sasimonthakan** [vdemichev/DiaNN#1124] [Q&A] [answered]
Hello,

Thank you very much for the amazing, free software!
For phosphoproteomics, is there a way I can visualize spectra of b/y ions that were used to identify phosphopeptides? I usually use this to validate my phosphoproteome with DDA, in addition to the site localization confidence. I tried the Viewer function but it only shows the intensity and RT, not the used spectra.

Thanks in advance,
Sasimonthakan

**Comment by vdemichev** [comment, marked as answer]
Hi Sasimonthakan,

Yes, both DIA-NN Viewer and Skyline (with MBR) should work fine for phosphoproteomics. 

> it only shows the intensity and RT

But you can see the extracged chromatrograms from all the fragments? DIA-NN makes the decision to report a particular identification based on those.

Best,
Vadim

**Reply by Sasimonthakan** [reply]
Thanks so much for your quick response.
Yes, but I also wanna look at the MS2 spectra. I found the tutorial on this video: https://www.youtube.com/watch?v=wadteC807_4.

Best,
Sasimonthakan

**Reply by vdemichev** [reply]
Basically, the hight of extracted chromatograms at a particular time point is the spectrum at that timepoint. Just the chromatogram visualisation is a lot more informative, if you see a nice chromatogram, you know the signal is not just noise.
